src/rwe/OpaqueId.h

```diff
@@ -23,6 +23,12 @@ namespace rwe
         {
             return !(rhs == *this);
         }
+
+        bool operator<(const OpaqueId& rhs) const
+        {
+            return value < rhs.value;
+        }
```

Comment:
I'm not _really_ sure if it makes sense to add this to OpaqueId - I never imagined IDs having a defined ordering. But I guess I can let it slide for this as it seems like not having it would make the solution a lot more of a hassle.

Reply:
yeah, OTA consistently tracks groups of units in build order, sorting the ids seemed the simplest way